ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/policy-maps-copp.j2

```diff
@@ -0,0 +1,24 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - policy map copp #}
+{% if policy_maps.copp_system_policy.enable is arista.avd.defined(true) %}
+!
+policy-map type copp copp-system-policy
+{%     for class in policy_maps.copp_system_policy.classes | arista.avd.natural_sort("name") %}
+   class {{ class.name }}
+{%         if class.rate_unit is arista.avd.defined %}
```

Comment:
Can you please check if we can use different `rate_unit`s for `shape` and `bandwidth`

For Example:
```
class hello
      shape kbps 22
      bandwidth pps 33
```

Reply:
If we can then please update the schema and template (eos and documentation).